### PR TITLE
shrink gluster volume

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat @nnDarshan @nthomas-redhat please review

**Comment by GowthamShanmugam** [comment]
verified

File: tendrl/gluster_integration/objects/volume/atoms/shrink/__init__.py [hunk 1]
```diff
@@ -96,20 +96,46 @@ def run(self):
                             lock.acquire(lock_ttl=60)
                     except etcd.EtcdLockExpired:
                         continue
+                subvolumes = NS._int.client.read(
```
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/shrink/__init__.py]: Here you are reading bricks and variable is called subvolume. Subvolume is actually a set of bricks.

Comment (GowthamShanmugam) [inline tendrl/gluster_integration/objects/volume/atoms/shrink/__init__.py]: @shtripat as per new structure clusters/{cid}/Volumes/{vid}//Bricks/{sub_vol_id}

so here i am reading subvolumes only., inside subvolumes brick details re present

Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/shrink/__init__.py]: Oh ok. LGTM otherwise

**Pull request closed without merging**